packages/dnb-eufemia/src/components/date-picker/__tests__/DatePicker.test.tsx

```diff
@@ -2575,6 +2580,461 @@ describe('DatePicker calc', () => {
       rightPicker.querySelector('.dnb-date-picker__header__title')
     ).toHaveTextContent('november 2024')
   })
+
+  describe('DatePicker date limit validation', () => {
```

Comment:
```suggestion
  describe('minDate and maxDate', () => {
```

Not necessarily better. But easier to read/locate the props.